modules/administration-guide/partials/proc_collecting-dev-workspace-operator-metrics-with-prometheus.adoc

```diff
@@ -86,24 +73,36 @@ data:
           tls_config:
             insecure_skip_verify: true
           static_configs:
-            - targets: ['devworkspace-webhookserver:9443'] <5>
+            - targets: ['devworkspace-webhookserver.__<DWO_{orch-namespace}>__:9443'] <5>
 ----
 <1> The rate at which a target is scraped.
 <2> The rate at which the recording and alerting rules are re-checked.
 <3> The resources that Prometheus monitors. In the default configuration, two jobs, `DevWorkspace` and `DevWorkspace webhooks`, scrape the time series data exposed by the `devworkspace-controller-metrics` and `devworkspace-webhookserver` Services.
-<4> The scrape metrics from port `8443`.
-<5> The scrape metrics from port `9443`.
+<4> Scrape metrics from port `8443`. Replace `__<DWO_namespace>__` with the {orch-namespace} where the `devworkspace-controller-metrics` `Service` is located.
```

Comment:
I recommend keeping `The` to make it clear that it's not used as a verb (in the imperative form) here and to align it with the previous lines that all start with `The`.